src/requires/pyplot.jl

```diff
@@ -72,8 +72,7 @@ function plot(flat::AbstractFourierLattice, Rs::DirectBasis{D};
               filling::Union{Real, Nothing}=0.5, 
               isoval::Union{Real, Nothing}=nothing,
               repeat::Union{Integer, Nothing}=nothing,
-              fig=nothing) where D
- 
+              fig=nothing, ax=nothing) where D
```

Comment:
Keep style of surrounding code:
```suggestion
              fig=nothing, 
              ax=nothing) where D

```